### PR TITLE
feat(relaseNote) connector removal

### PR DESCRIPTION
**Pull request by rbioteau**
Closes [BS-16818](https://bonitasoft.atlassian.net/browse/BS-16818)

### CODE REVIEW

**Comment by rbioteau** [comment]
http://doc.rd.lan:50394/7.6?page=release-notes#toc11

File: md/release-notes.md [hunk 1]
```diff
@@ -54,7 +54,20 @@ __Note__: the migration process automatically removes this file from an existing
 
 
 ## Breaking changes
-Change method updateGroup() in identityAPI to forbid group name update while new name already exist.
+
+### API
+* Change method [updateGroup()](https://documentation.bonitasoft.com/javadoc/api/7.6/org/bonitasoft/engine/api/GroupAPI.html#updateGroup-long-org.bonitasoft.engine.identity.GroupUpdater-) in identityAPI to forbid group name update while new name already exist.
```
Comment (NathalieC) [inline md/release-notes.md]: quick fix: "when new name already exists. "
What is this fix's BS?

File: md/release-notes.md [hunk 2]
```diff
@@ -54,7 +54,20 @@ __Note__: the migration process automatically removes this file from an existing
...
+
+### Connectors
+* SugarCRM version API v4 has been **removed**: newer versions of [SugarCRM expose a REST API](http://support.sugarcrm.com/Documentation/) that should be used with the REST connector.
+* [SAP Connector (jco2)](sap-jco-2.md) has been **removed from Community edition** due license incompatibility.
+* Talend connectors has been **removed**: newer versions of [Talend expose a REST API](https://help.talend.com/reader/ISPDm8GQ6s0HN0348QulWg/HF8MMjUq3bllDlzOz2lqxw) that should be used with the REST connector.
+* Google Calendar v2 connectors has been **removed**: v2 API are not more supported by Google, use Google Calendar v3 instead.
+
+::: info
+**Migration**: in order to migrate proceses using those connectors, you wil have to export those connectors from a previous Bonita Studio version and re-import them.
```
Comment (NathalieC) [inline md/release-notes.md]: typos: 
- processes 
- will

**Review by NathalieC — COMMENTED** [review]
a few typos

**Pull request merged**